ParabolInc/parabol
# feat: implement azure task integration

Fix https://github.com/ParabolInc/parabol/issues/6970

### To test 

- [ ] If you haven't added any integration, open the task footer menu, see the "Add Azure" menu item
- [ ] Clicking on the "Add Azure" menu item successfully integrates with Azure
- [ ] Open the task footer and see that Azure menu items are populated in the menu
- [ ] Click on a menu item and see that the task successfully integrates, creating an issue in the selected project
- [ ] User A creates a task. User B integrates the task with Azure. See that there's a comment in the Azure issue saying that it was created by User A and integrated by User B

Note: I need to refresh the Azure token ~once per day or it expires. See issue: https://github.com/ParabolInc/parabol/issues/7006
